src/electron/emain.ts

```diff
@@ -717,6 +747,7 @@ function runActiveTimer() {
     setTimeout(runActiveTimer, 5000); // start active timer, wait 5s just to be safe
     await app.whenReady();
     await createMainWindowWrap();
+    reregisterGlobalShortcut("Cmd+F4");
```

Comment:
How does this play on Linux? I think we should add an OS check here

Reply:
Also will this reset on start every time?

Reply:
oops, i had two commits not pushed to github.  that Cmd+F4 was just testing (removed in the final).  I also have a check for linux which switches between Cmd (for MacOS), and Alt (for Linux/Windows).